opensrp-chw-anc/src/main/java/org/smartregister/chw/anc/fragment/BaseAncRespondersCallDialogFragment.java

```diff
@@ -54,13 +67,26 @@ public View onCreateView(final LayoutInflater inflater, final ViewGroup containe
     }
 
     private void initUI(ViewGroup rootView) {
-        if (StringUtils.isNotBlank(ancResponderPhoneNumber)) {
+        if (isResponder && StringUtils.isNotBlank(ancResponderPhoneNumber)) {
             TextView ancWomanNameTextView = rootView.findViewById(R.id.responder_name);
             ancWomanNameTextView.setText(ancResponderName);
             TextView ancCallAncWomanPhone = rootView.findViewById(R.id.anc_call_responder_phone);
             ancCallAncWomanPhone.setTag(ancResponderPhoneNumber);
             ancCallAncWomanPhone.setText(getName(getCurrentContext().getString(R.string.anc_call), ancResponderPhoneNumber));
             ancCallAncWomanPhone.setOnClickListener(listener);
```

Comment:
Quick question, are these the ANC Woman name and phone text details or the responder's details?